modules/running-your-dynamic-plug-in.adoc

```diff
@@ -51,7 +51,7 @@ $ yarn http-server -a <server name>
 +
 [source,terminal]
 ----
-$ ./bin/bridge -plugins console-demo-plugin=http://localhost:9001/
+$ yarn run start-console
```

Comment:
This is only needed for local development; and would be outside the scope of our support for customers (correct)? 

https://access.redhat.com/support/offerings/production/soc/ says Code development is outside of support.

Reply:
The whole point is for Red Hat teams, customers, and partners to be able to develop their own plugins. We don't have to support their code, but we need to give them the tools to be able to write their own plugins. You would never run the template without making changes since it's just a hello world example.